Insurance_company/accounts/models.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.core.validators import RegexValidator
+from django.db import models
+
+User = settings.AUTH_USER_MODEL
+
+
+class Customer(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    pesel = models.CharField(max_length=11, unique=True, validators=[RegexValidator(regex='^\\d{11}$', message='PESEL must contain 11 digits')])
```

Comment:
tu trzeba pamiętać, że tali walidator zostanie użyty jeśli skorzystamy z modelforma, a tez niżej jest inna walidacja do tego samego to nie wiem czy bym z tego nie zrezygnował

Reply:
aa rozumiem oki, pomyślałam że dodatkowa walidacja ilości cyfr w tym miejscu, byłaby ok, ale tez przeszło mi przez myśl, że suma kontrolna mogłaby w zasadzie każdą walidację załatwić. Wobec tego też zrezygnuję, dziękuje za cenna uwagę.